docs/Brew-Livecheck.md

```diff
@@ -62,10 +62,10 @@ When in doubt, start with one of these examples instead of copy-pasting a `livec
 ### File names
 
 ```ruby
-  livecheck do
-    url "https://www.example.com/downloads/"
-    regex(/href=.*?example[._-]v?(\d+(?:\.\d+)+)\.t/i)
-  end
+livecheck do
+  url "https://www.example.com/downloads/"
+  regex(/href=.*?example[._-]v?(\d+(?:\.\d+)+)\.t/i)
+end
```

Comment:
The reason for the indentation here is so that you can easily copy and paste this text into a formula/cask. Depending on your text editor, if there's no leading spaces here you have to manually indent the lines whenever you copy and paste it. There's no harm in having the spaces but removing them would make it less beneficial, so I think it should be kept like this to make it easy for contributors/maintainers.

Reply:
Running `brew style --fix` is a required step for both `homebrew/core` and `homebrew/cask`. If you are using an editor which doesn't automatically indent this, you will also likely introduce some sort of other whitespace error, so `brew style --fix` is necessary in most cases anyways.